tac/governing-documents/release-signing.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+title:  Release Signing
+parent: Governing Documents
+grand_parent: LF Decentralized Trust TAC
+nav_order: 1
+---
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# Securing and Verifying Releases
+
+The LF Decentralized Trust (LFDT) emphasizes a structured approach to governance, particularly in the context of artifact signing for its projects. Artifacts, which are essential outputs from each release, include binaries, container images, checksum files, documentation, provenance metadata, and software bills of materials (SBOM).
+
+LFDT has a holistic approach to governing the security of projects and implements the highest levels of standards. The objective is to establish a consistent and secure approach to signing artifacts, leveraging industry-standard tools and practices while allowing flexibility for individual project implementations. Software verification processes are made available to ensure the integrity and authenticity of all artifacts throughout their lifecycle. A monitoring framework is introduced to score the project, rating the release process.
+
+This initiative aligns with efforts to establish a secure software supply chain for LFDT projects and achieve full [SLSA (Supply Chain Levels for Software Artifacts) compliance](https://slsa.dev/spec/v1.0/levels) in the release process.
+
+This governance framework prioritizes understanding the "what" of artifact signing, which includes:
+- The importance of signing artifacts to ensure their integrity and authenticity.
+- The necessity of meeting security standards, such as SLSA compliance.
+
+The technical "how" of implementing will be left to the discretion of individual projects, allowing for flexibility and innovation.
+
+## LFDT Project Compliance
+
+1. The LFDT Technical Advisory Committee (TAC) standardizes the signing of artifacts using tools from the Linux Foundation’s digital signing project, [OpenSSF Sigstore](https://openssf.org/projects/sigstore/).
+2. Each project must document the process of verifying the artifacts using [OpenSSF Sigstore](https://openssf.org/projects/sigstore/). The project must have a file called `RELEASE.md`.
+3. Each project must enable and apply for a Sigstore's scorecard. The project must capture the scorecard's result and generate a badge visible on the README.md file.
+4. An LFDT incubation project will follow at least SLSA Level 1, while graduated projects will meet at least SLSA Level 2.
```

Comment:
```suggestion
4. An LFDT incubation project must meet at least SLSA Level 1, while graduated projects must meet at least SLSA Level 2.
```

Reply:
I've gone back and forth on whether graduated projects MUST sign artifacts, I think that is the main item to discuss.